pyVmomi/SoapAdapter.py

```diff
@@ -1384,6 +1384,11 @@ def _CloseIdleConnections(self):
 
          for conn, _ in idleConnections:
             self._CloseConnection(conn)
+      else:
```

Comment:
This means we always close connection when the timeout is negative? But negative supposed to mean we never close connection. I agree that never close connection is a problem when the session times out for those existing connections. Maybe we should never allow a negative value?